webassembly/example.html

```diff
@@ -1,51 +1,168 @@
 <!DOCTYPE html>
-<html>
+<html data-theme="dark">
```

Comment:
Good idea! It would require to add a widget somewhere in the page and add a javascript snippet to change the value of `data-theme`. Not in this PR though.

Reply:
ok